projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -69,6 +69,10 @@ public static function init() {
 		// Load the Newsletter category settings.
 		add_action( 'enqueue_block_assets', array( __CLASS__, 'load_newsletter_categories_settings' ), 999 );
 
+		// Remove WordPress Events and News feed widget from the dashboard.
```

Comment:
I think the "trend" now is to write new mu-wpcom features in a separate feature file, so that features are split logically into individual files. I can help refactor it to a `wpcom-admin-dashboard` feature today 👍